GeoJSON/geojson-encoding-rule.md

```diff
@@ -0,0 +1,270 @@
+| GM_Curve          | LineString | In GML, Curves can also be nonlinear segments or arcs. GeoJSON only supports linear segments. |
+| GM_MultiCurve     | MultiLineString | In GML, Curves can also be nonlinear segments or arcs. GeoJSON only supports linear segments. |
+| GM_MultiPoint     | MultiPoint |  |
+| GM_MultiPrimitive | not supported | `GM_MultiPrimitive` is an abstract type. |
+| GM_MultiSurface   | MultiPolygon |  |
+| GM_Object         | Any GeoJSON Geometry type | `GM_Object` is an abstract type, any GeoJSON geometry can be used. |
+| GM_Point          | Point |  |
+| GM_PolyhedralSurface | not supported | At this point, this specification does not support 3D meshes. |
+| GM_Primitive      | Any GeoJSON Geometry type | `GM_Primitive` is an abstract type, any GeoJSON geometry can be used. |
+| GM_Surface        | Polygon | A `GM_Surface` can have many SurfacePatches, it is thus mapped to MultiPolygon. |
+| GM_Tin            | not supported | A TIN without triangulation could be converted to a MultiPoint object. |
+| GM_Triangle       | Polygon |  |
+
+Where a class has one attribute with a geometry type, this attribute will be mapped to the `geometry` property in GeoJSON, while all other properties (attributes and association roles) will be mapped to properties inside the `properties`. A theme-specific profile has to define which geometry property is the "default" geometry that should be mapped to the `geometry` property in GeoJSON.
+
+#### ISO 19108 - Temporal types
+
+For types from ISO 19108 used in INSPIRE schemas, suitable mappings need to be found on a case-by-case basis. The default should be to use the [Simple Period](/model-transformations/SimplePeriod.md) substitution rule.
+
+#### ISO 19115 - Metadata types
+
+For types from ISO 19115 used in INSPIRE schemas, suitable mappings need to be found on a case-by-case basis. For `CI_Citation`, the [Simple Citation](/model-transformations/SimpleCitation.md) substitution rule can be applied.
+
+#### Abstract Types as property types
+
+Where an abstract type with multiple concrete sub-types is used as a property type, a suitable choice of a concrete subtype should be made on a case-by-case basis, with the objective of limiting the potential geometry types that can occur and to make processing easier.
+
+#### Union Types
+
+A `union` represents a choice between multiple properties with potentially different value types, such as in the AreaOfResponsibilityType, where there are options such as `areaOfResponsibilityByAdministrativeUnit` and `areaOfResponsibilityByNamedPlace`. `Reference` and a `Building`. The multiplicity of these options may also differ.
+
+For Union Types used in INSPIRE schemas, suitable mappings need to be found on a case-by-case basis.
+
+#### Enumerations and Code Lists
+
+Properties that represent values form code lists are encoded using the `SimpleCodelistReference` [rule described here](/model-transformations/SimpleCodelistReference.md).
```

Comment:
Move this into the Properties section (as code list/enumeration-valued properties)?